automatic/1password/update.ps1

```diff
@@ -15,36 +14,50 @@ function global:au_SearchReplace {
   }
 }
 
-function global:au_AfterUpdate {
+function global:au_BeforeUpdate {
   . "$PSScriptRoot/update_helper.ps1"
-  if ($Latest.PackageName -eq '1password4') {
     removeDependencies ".\*.nuspec"
-  } else {
+}
+
+function global:au_AfterUpdate {
+  . "$PSScriptRoot/update_helper.ps1"
+  if ($Latest.PackageName -eq '1password') {
     addDependency ".\*.nuspec" 'dotnet4.6.1' '4.6.01055.20170308'
-  }
+  } 
 }
 
 function global:au_GetLatest {
-  $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
 
   $streams = @{}
-  $res = @('OPW4';'format=zip$')
+  $res = @('OPWY';'OPW4';'OPW7')
   $res | % {
     $re = $_
-    $url32 = $download_page.Links | ? href -match $re | select -First 1 -expand href
+    switch -w ( $re ) {
+        'OPW4' {
+            $url32 = 'https://app-updates.agilebits.com/download/OPW4'
+			$kind = '1password4'
+        }
+        'OPW7' {
+            $url32 = 'https://app-updates.agilebits.com/download/OPW7'
+			$kind = '1password'
+        }
+        'OPWY' {
+            $url32 = 'https://app-updates.agilebits.com/download/OPW7/Y'
+			$kind = '1password'
+        }
+    }
     $url32 = Get-RedirectedUrl $url32
-
     $verRe = '[-]|\.exe$'
     $version = $url32 -split $verRe | select -last 1 -skip 1
+    $version = $version -replace('\.BETA',' beta')
```

Comment:
I don't see any how the urls actually being beta urls.
What's the format of the version from before this replacement is used?
`7.3.5.BETA`? If so, it's okay, but if not I think we'll remove this parsing for now, and wait until it's needed.

Reply:
Yes, the download URL(s)
1. `https://c.1password.com/dist/1P/win6/1PasswordSetup-7.1.566.BETA.exe`
2. `https://c.1password.com/dist/1P/win6/1PasswordSetup-7.0.558.BETA.exe`
3. `https://c.1password.com/dist/1P/win6/1PasswordSetup-7.0.554.BETA.exe`
4. `https://c.1password.com/dist/1P/win6/1PasswordSetup-7.0.543.BETA.exe`
for the previous beta(s)